ChangeLog

```diff
@@ -1,7 +1,7 @@
 2015-05-17  Matt P. Dziubinski  <matdzb@gmail.com>
 
         * inst/include/Rcpp/platform/compiler.h: Added RCPP_USING_CXX0X_OR_LATER
-        * inst/include/Rcpp/sugar/functions/sapply.h: Updated to support lamda functions
+        * inst/include/Rcpp/sugar/functions/sapply.h: Updated to support lambda functions
```

Comment:
Yeah, haven't noticed it at first, either! :D Must be the phonetic pronunciation thing... ;-)